src/ViewModels/test/CfInstanceViewModelTests.cs

```diff
@@ -25,6 +27,14 @@ public void TestInit()
             {
                 _receivedEvents.Add(e.PropertyName);
             };
+
+            MockUiDispatcherService.Setup(mock => mock.
+                RunOnUiThread(It.IsAny<Action>()))
+                    .Callback<Action>(method =>
```

Comment:
nit:  "action" seems more appropriate then "method"

Reply:
Fair - I think "action" is technically more accurate